doc/themes/scikit-learn/static/nature.css_t

```diff
@@ -219,10 +219,6 @@ div.navbar div.nav-icon {
     width: 50px;
 }
 
-.gsc-branding {
-    display: none;
```

Comment:
In order to remove "Powered by Google", keep this section and add `!important` at the end of this line. From what I can gather Google his adding its own CSS dynamically and we want our own CSS to prevail. Adding `!important` is one way to ensure that.